woocommerce/woocommerce-gateway-stripe
# refactor: UPE banner content

# Changes proposed in this Pull Request:

A few things about the UPE opt-in banner:
- The banner is now enqueued _only_ on the payment methods settings pages (and not on _all_ WC admin pages)
- The banner's content is now different between the settings pages and the main Stripe page (confirmed with UX)
- Adjusted the spacing and styles to more closely align to the design

Tested in Chrome, Safari, Firefox.

![2021-09-09 17 12 52](https://user-images.githubusercontent.com/273592/132769458-b5c40c0b-2994-4e88-a494-9be55db9786f.gif)

![2021-09-09 17 36 22](https://user-images.githubusercontent.com/273592/132771550-c140911c-c81d-49af-9b5a-c05ffcf7789b.gif)



# Testing instructions
- Ensure the `_wcstripe_feature_upe_settings` flag is enabled
- Ensure the `woocommerce_stripe_settings.upe_checkout_experience_enabled` flag is disabled
- Go to the main Stripe page: http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe
- The banner's content is now different:
![Screen Shot 2021-09-09 at 5 08 48 PM](https://user-images.githubusercontent.com/273592/132769002-7ba4c08e-0fd6-4f3e-b30f-4e960e5fea56.png)
- On the other gateway pages, the banner is similar (but the content now dynamically includes the payment method's title)
![Markup 2021-09-09 at 17 09 58](https://user-images.githubusercontent.com/273592/132769127-1ddfbf17-2520-4544-87c2-cfde3d28e69e.png)
- Examples:
  - http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe_sepa
  - http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe_giropay
  - http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe_eps
  - http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe_bancontact
  - http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe_sofort
  - http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe_p24
  - http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe_alipay
  - http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe_multibanco
  - http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe_ideal
